files/zh-cn/web/api/element/insertadjacentelement/index.md

```diff
@@ -50,28 +48,21 @@ element.insertAdjacentElement(position, element);
 <!-- afterend -->
 ```
 
-> **备注：** 当节点处于 DOM 树中而且有一个父元素的时候 `beforebegin` 和 `afterend` 操作才能起作用。
+> **备注：** 只有当节点处于 DOM 树中且有一个父元素时 `beforebegin` 和 `afterend` 才会生效。
 
 ## 例子
 
 ```js
-beforeBtn.addEventListener("click", function () {
-  var tempDiv = document.createElement("div");
-  tempDiv.style.backgroundColor = randomColor();
-  activeElem.insertAdjacentElement("beforebegin", tempDiv);
-  setListener(tempDiv);
-});
-
-afterBtn.addEventListener("click", function () {
-  var tempDiv = document.createElement("div");
-  tempDiv.style.backgroundColor = randomColor();
-  activeElem.insertAdjacentElement("afterend", tempDiv);
-  setListener(tempDiv);
-});
+// 原为 <div id="one">one</div>
+const one = document.getElementById("one");
+const two = document.createElement("div");
+two.innerText = "two";
+one.insertAdjacentHTML("afterend", two);
+
+// 此时，新结构变为：
+// <div id="one">one</div><div id="two">two</div>
 ```
 
-看看我们在 Github（也可以参考 [源码](https://github.com/mdn/insert-adjacent/blob/master/insertAdjacentElement.html)）上的 [insertAdjacentElement.html](http://mdn.github.io/insert-adjacent/insertAdjacentElement.html) 演示。在一个容器当中我们有一组{{htmlelement("div")}} 元素。点击其中一个 div 时，这个容器会处于选中状态，然后你就可以按下*Insert before* 或*Insert after* 按钮通过 `insertAdjacentElement() 方法`来把一个新的 divs 插入到选中的元素前面或者后面。
-
```

Comment:
The demo was removed, I recommend updating the upstream. /cc @yin1999

Reply:
The demo is listed in upstream: https://github.com/mdn/content/blob/4b600e6fa6ccae366aa6c885240f60b748c05c05/files/en-us/web/api/element/insertadjacentelement/index.md?plain=1#L90-L94

What do you mean here?

Reply:
Oops, that is my fault. I tried directly to visit the link but got 404. I didn't realize the link was updated in the original text.

Hi, @xiyuesaves. Please add the line#73 back and update the link and the translation. 😊

Reply:
> Oops, that is my fault. I tried directly to visit the link but got 404. I didn't realize the link was updated in the original text.
> 
> Hi, @xiyuesaves. Please add the line#73 back and update the link and the translation. 😊

Okay, thank you for your help 😊. I'm still not very familiar with this process.